### ISSUE TITLE
PLY file with name property normalx not automatically parsed

### ISSUE THREAD

**Issue by jo-chemla** [CloudCompare/CloudCompare#1846]
**Describe the bug**
Hi again,
I've got a ply file whose properties contains a field per coordinate for normals named `normalx, normaly, normalz`. 
I can see in the [PLY filter](https://github.com/CloudCompare/CloudCompare/blob/05d50e5b580f298cc2762a7e73c8da67ab694031/libs/qCC_io/src/PlyFilter.cpp#L1132) that it should automagically parse these to the points normals on import via this line.

```cpp
if (nxIndex == 0 && (propName.contains("NX") || (elementName.contains("NORM") && propName.endsWith("X"))))
	nxIndex = i;
``` 

However, I have to manually select them on import via the dialog dropbox:
![image](https://github.com/CloudCompare/CloudCompare/assets/16822841/31b40c07-ecb5-4219-ab3f-4263cca1bf83)

**How to reproduce**
Load a file whose normals are stored as ply like [this sample](https://www.dropbox.com/scl/fi/70gz75n3errkcbj7jy7ig/ply-with-normals-bin-pdal-out.ply?rlkey=kcxnh8wnoy6on34uahjkmu433&dl=0)

**Expected behaviour**
Should automatically map ply normal fields named `normalx, normaly, normalz` to the corresponding normals properties of the ptc vertices.

**Additional context**
This should do the trick, adding another test-case where we check also if `propName` starts with `NORM`
```cpp
if (nxIndex == 0 && (propName.contains("NX") || (elementName.contains("NORM") && propName.endsWith("X")) || (propName.contains("NORM") && propName.endsWith("X"))))
```

**Comment by dgirardeau** [comment]
Ah, I see, it's the property that contains 'NORM' here, not the element name.

I've added the support for this case. It will be soon available (both in Github and in the latest 2.13.beta version)

**Comment by jo-chemla** [comment]
Thanks, very much appreciated!
